src/tsconfig-base.json

```diff
@@ -16,7 +16,7 @@
     "noImplicitAny": false,
     "newLine": "lf",
     "lib": ["ES2020"],
-    "typeRoots": ["./node_modules/@types", "src/@types"],
+    "typeRoots": ["../node_modules/@types"],
```

Comment:
I believe `typeRoots` is redundant now that you're not referencing something other than a `**/node_modules/@types`: https://www.typescriptlang.org/tsconfig#typeRoots

Reply:
appears you are correct! thanks!!!